Product/DBScripts/pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <groupId>org.connectopensource</groupId>
         <artifactId>Product</artifactId>
-        <version>5.0-SNAPSHOT</version>
+        <version>5.0.0-RC1</version>
```

Comment:
Shouldn't this be a SNAPSHOT version, since code changes are made in this pull request?